website/docs/getting-started/using.md

```diff
@@ -2,16 +2,18 @@
 title: Using
 ---
 
-To use the Hook, first call it, telling it how many items your menu will have:
+To use the Hook, first call it, telling it how many items your menu will have along with a valid HTMLElement subtype:
 
 ```jsx
-const { buttonProps, itemProps, isOpen } = useDropdownMenu(numberOfItems);
+const { buttonProps, itemProps, isOpen } = useDropdownMenu<HTMLSubtypeElement>(numberOfItems);
 ```
 
-Take the `buttonProps` object and spread it onto a button:
+Take the `buttonProps` object and spread it onto the corresponding HTML element to your subtype:
 
 ```jsx
 <button {...buttonProps}>Example</button>
+
+<div {...buttonProps}>Example</div>
```

Comment:
These two lines can be removed, there only needs to be one example of spreading the button props.